hphp/runtime/vm/jit/vasm-arm.cpp

```diff
@@ -1543,9 +1531,7 @@ void lower(const VLS& e, movzlq& i, Vlabel b, size_t z) {
 
 void lower(const VLS& e, movtdb& i, Vlabel b, size_t z) {
   lower_impl(e.unit, b, z, [&] (Vout& v) {
-    auto d = v.makeReg();
-    v << copy{i.s, d};
-    v << movtqb{d, i.d};
+    v << copy{i.s, i.d};
```

Comment:
@dave-estes - Hi Dave,  I'm not sure.  If your new version goes in first, I'll have to retest.